### ISSUE TITLE
Tests don't pass, babel-jscs released a breaking change with 5.7.0

### ISSUE THREAD

**Issue by elicwhite** [jscs-dev/babel-jscs#18]
Original post on the JSCS repo here: https://github.com/jscs-dev/node-jscs/issues/1567

When I run babel-jscs on my non es6 code, it fails while parsing every file. 

Digging a little bit deeper...

index#attachComments is called with >900 tokens, and fails on this line:

```
} else if (firstComment.start < tokens[0].range[0]) {
```

If I print tokens[0], I get

```
{ type: 'String',
  value: 'use strict',
  start: 24,
  end: 36,
  loc: { start: { line: 3, column: 0 }, end: { line: 3, column: 12 } } }
```

It doesn't have a range, which is why it fails when trying to read index 0 from the undefined property.

**Comment by elicwhite** [comment]
It looks like babel-core updated from 5.6 to 5.7 and had breaking changes. Looking further...

**Comment by elicwhite** [comment]
Reading their github comparision between 5.6.14 and 5.7.0 is impossible because all the files were moved around. I'm not really sure how you want to proceed with this.

**Comment by hzoo** [comment]
Since I'm working with babel-eslint as well I will probably see what breaks from that repo - @sebmck just changed a few things with the parser that may break some stuff so just need to make similar changes - this time related to the `range` property.

**Comment by mrjoelkemp** [comment]
Thanks @hzoo. This is holding up the 2.0 release. So landing #19 seems like the best move for now.

**Comment by hzoo** [comment]
Ok made `2.0.3` with fixes - I just pinned babel-core to `~5.8.3` for now although we could probably just do ^ like before
